plutus-core/plutus-core/src/PlutusCore/Builtin/Result.hs

```diff
@@ -184,28 +126,25 @@ variable).
 --
 -- This is useful for providing 'AsUnliftingError' instances for types such as 'CkUserError' and
 -- 'CekUserError'.
-_UnliftingErrorVia :: Pretty err => err -> Prism' err UnliftingError
-_UnliftingErrorVia err = iso (MkUnliftingError . display) (const err)
-{-# INLINE _UnliftingErrorVia #-}
-
--- | See Note [Structural vs operational errors within builtins]
-_StructuralUnliftingError :: AsBuiltinError err => Prism' err UnliftingError
-_StructuralUnliftingError = _BuiltinUnliftingEvaluationError . _StructuralError
-{-# INLINE _StructuralUnliftingError #-}
-
--- | See Note [Structural vs operational errors within builtins]
-_OperationalUnliftingError :: AsBuiltinError err => Prism' err UnliftingError
-_OperationalUnliftingError = _BuiltinUnliftingEvaluationError . _OperationalError
-{-# INLINE _OperationalUnliftingError #-}
 
 throwNotAConstant :: MonadError BuiltinError m => m void
-throwNotAConstant = throwing _StructuralUnliftingError "Not a constant"
+throwNotAConstant = throwError (BuiltinUnliftingEvaluationError $ MkUnliftingEvaluationError $ StructuralError $ MkUnliftingError "Not a constant")
```

Comment:
Same sentiment here: don't see much value in factoring `throwError` in.

It feels like syntactic win (`throw` prefix is shorter than `throwError .`) IMO doesn't justify the loss of utility (`notAConstantError :: BuiltinError` is potentially usable without throwing)